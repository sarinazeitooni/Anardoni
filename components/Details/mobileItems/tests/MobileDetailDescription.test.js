import React from 'react';
import { shallow } from 'enzyme';
import MobileDetailDiscription from "../MobileDetailDescription";
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
describe("MobileDetailDiscription component" , ()=>{
    const initialState = {}
    const mockStore = configureStore();
    let store = mockStore(initialState);
    const container = shallow(<Provider store={store}><MobileDetailDiscription item={[]}/></Provider>);
    it("it should render a div",()=>{
        expect(container).toMatchSnapshot();
    });
    it("it should render a mobile-card-container" ,()=>{
        expect(container.find('div.more-container')).toBeTruthy();
    });
});