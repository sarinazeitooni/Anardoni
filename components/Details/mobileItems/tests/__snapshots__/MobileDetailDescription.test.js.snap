// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MobileDetailDiscription component it should render a div 1`] = `ShallowWrapper {}`;
